From cc4e5ec281391db709ddb607dfb937db71dbac2e Mon Sep 17 00:00:00 2001
From: Marko Kaznovac <kaznovac@users.noreply.github.com>
Date: Thu, 10 Jan 2019 15:00:56 +0100
Subject: [PATCH 1/1] add compile script call to composer.json

expose call for compiling `composer.phar` in composer json, along with description
---
 composer.json | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/composer.json b/composer.json
index 41048903..5ed96996 100644
--- a/composer.json
+++ b/composer.json
@@ -72,8 +72,13 @@
         "bin/composer"
     ],
     "scripts": {
+        "compile": "@php -dphar.readonly=0 bin/compile",
         "test": "phpunit"
     },
+    "scripts-descriptions": {
+        "compile": "Compile composer.phar",
+        "test": "Run all tests"
+    },
     "support": {
         "issues": "https://github.com/composer/composer/issues",
         "irc": "irc://irc.freenode.org/composer"
-- 
2.21.0

